### PR TITLE
Bring privatebin up to date with source repo

### PR DESCRIPTION
**Pull request by philmcmahon**
Just one big monster pr to bring guardian/Privatebin into the future - we are currently using version 1.3.3, latest is 1.5.2

See https://github.com/guardian/deploy-tools-platform/blob/main/cloudformation/privatebin.template.yaml#L358 for where we check out this repo

### CODE REVIEW

File: .github/workflows/snyk-scan.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# This is a basic workflow to help you get started with Actions
```
Comment (akash1810) [inline .github/workflows/snyk-scan.yml:1]: Does this file clash with [our own](https://github.com/guardian/PrivateBin/blob/master/.github/workflows/snyk.yml)?

**Review by akash1810 — COMMENTED** [review]
A blocking comment on the Snyk workflow, otherwise LGTM.

We should also rollout the [containerised version](https://github.com/guardian/privatebin-docker) of this, which reduces a lot of maintenance work. The project is currently paused, awaiting a refactor to GuCDK.

File: .github/workflows/snyk-scan.yml (hunk 1 above)
Comment (philmcmahon) [reply .github/workflows/snyk-scan.yml:1]: Good spot! I have a branch without this file in it but don't have push access to this repo, would you mind adding me? Or my team digital investigations

Comment (akash1810) [reply .github/workflows/snyk-scan.yml:1]: Added https://github.com/orgs/guardian/teams/guardian-developers-write with write permissions now.

**Comment by philmcmahon** [comment]
Replaced by https://github.com/guardian/PrivateBin/pull/3

File: .github/workflows/snyk-scan.yml (hunk 1 above)
Comment (philmcmahon) [reply .github/workflows/snyk-scan.yml:1]: thank you!

**Pull request closed without merging**